cmd/ndm_daemonset/app/command/commands.go

```diff
@@ -17,24 +17,14 @@ limitations under the License.
 package command
 
 import (
-	"flag"
+	goflag "flag"
 
 	"github.com/golang/glog"
 	"github.com/openebs/node-disk-manager/pkg/util"
 	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
 )
 
-// NodeDiskManagerOptions defines a type for the options of NDM
-type NodeDiskManagerOptions struct {
```

Comment:
All the usages of `NodeDiskManagerOptions` also need to removed, since we are not using it.

Reply:
Removed the references to `NodeDiskManagerOptions` as part of https://github.com/openebs/node-disk-manager/pull/325/commits/dc4751cb1ad1e0ae3f3c44080879ca6cc57eeab3